src/meta/processors/job/JobManager.cpp

```diff
@@ -499,6 +503,9 @@ JobManager::showJob(JobID iJob) {
         return nebula::error(optJobRet);
       }
       auto optJob = nebula::value(optJobRet);
+      if (optJob.getParas().back() != spaceName) {
+        return nebula::cpp2::ErrorCode::E_JOB_NOT_IN_SPACE;
```

Comment:
Add this errorcode to MetaClient::handleResponse

Reply:
down